lib/install.js

```diff
@@ -1,43 +1,45 @@
 'use strict';
 
 var fs = require('fs');
-var mkdirp = require('mkdirp');
 var path = require('path');
 var async = require('async');
+var rimraf = require('rimraf');
 var cli = require('./cli');
 var config = require('./config');
-var driverUtils = require('./driver-utils');
+var initDriver = require('./init-driver');
 var configureIe = require('./configure-ie');
 
-Object.keys(config.binaries).forEach(function (resource) {
-  mkdirp.sync(config.binaries[resource].binary.path);
-});
+try {
+  // reset drivers if any critical config has changed (such as arch version).
```

Comment:
Arch change can occur if you set `SELENIUM_BINARIES_FORCE_32BITS` to true. I didn't find any reliable solution to determine if a binary is 32 or 64 bits. I chose to delete all drivers if this parameter has changed since the last installation.

If you already have the selenium-binaries package but you switch your env var to force 32 bits, you'll need to delete current binaries otherwise the package will not update your old 64 bits binaries.